geo/d2v-geo-common/src/main/kotlin/io/data2viz/geo/geometry/PolygonContains.kt

```diff
@@ -83,6 +83,12 @@ fun polygonContains(polygon: List<List<DoubleArray>>, point: DoubleArray): Boole
                 intersectionD2 /= intersectionNormalize
 
                 val phiArc = (if (antimeridian xor (delta >= 0)) -1 else 1) * asin(intersectionD2)
+
+                val pole = sin(phi);
+
+                // For next line see d3-geo issue #105
```

Comment:
When referencing anything, we can use links to save time. In that case the link is: 
https://github.com/d3/d3-geo/issues/105